lib/src/lib.rs

```diff
@@ -219,3 +219,72 @@ pub enum Error {
 	#[error("{0}")]
 	Api(#[from] crate::error::Api),
 }
+
+#[allow(dead_code)]
+pub(crate) static FFLAGS: FFlags = FFlags {
+    change_feed_live_queries: FFlagEnabledStatus {
+        enabled_release: false,
+        enabled_debug: false,
+        enabled_test: false,
+		env_override: "SURREALDB_CHANGE_FEED_LIVE_QUERIES",
+        owner: "Hugh Kaznowski",
+        description: "Disables live queries as a separate feature and moves to using change feeds as the underlying mechanism",
+        date_enabled_test: None,
+        date_enabled_debug: None,
+        date_enabled_release: None,
+        release_version: None,
+    }
+};
+
+#[derive(Clone, Copy, Debug, Eq, PartialEq, Ord, PartialOrd, Hash)]
+#[non_exhaustive]
+#[allow(dead_code)]
+pub(crate) struct FFlags {
+	change_feed_live_queries: FFlagEnabledStatus,
+}
+
+/// This struct is not used in the implementation;
+/// All the fields are here as information for people investigating the feature flag.
+#[allow(dead_code)]
+#[derive(Clone, Copy, Debug, Eq, PartialEq, Ord, PartialOrd, Hash)]
+pub(crate) struct FFlagEnabledStatus {
+	pub(crate) enabled_release: bool,
+	pub(crate) enabled_debug: bool,
+	pub(crate) enabled_test: bool,
+	pub(crate) owner: &'static str,
+	pub(crate) description: &'static str,
+	pub(crate) env_override: &'static str,
+	pub(crate) date_enabled_test: Option<&'static str>,
+	pub(crate) date_enabled_debug: Option<&'static str>,
+	pub(crate) date_enabled_release: Option<&'static str>,
+	pub(crate) release_version: Option<&'static str>,
+}
+
+impl FFlagEnabledStatus {
+	#[allow(dead_code)]
+	pub(crate) fn enabled(&self) -> bool {
+		let mut enabled = false;
+		if let Ok(env_var) = std::env::var(self.env_override) {
+			if env_var == "0" || env_var == "false" {
+				return false;
+			}
+			return true;
```

Comment:
This can cause surprising bugs. Why not actually check for `1` and `true` like we do for the false options? Someone could put garbage in there and we will interpret that as true.

Reply:
Perhaps it should be `1 / "" / "true"`?

Reply:
> Someone could put garbage in there and we will interpret that as true.
That is the intention. Normally, this feature will be off. I expect there to be a minimal number of days while we test if this feature is working. So setting it will usually indicate an override - i.e. something is broken and we want to revert it. We could improve this further, but if the env is set I do want it to default to true.

Reply:
> I believe it is like this so you can pass on an envvar without a value to enable the feature flag

Why though? An empty string is typically associated with false. Most if not all the time, users would just need to set that as `SURREALDB_CHANGE_FEED_LIVE_QUERIES=true`. I don't even see any reason to allow even `1` here because it's not descriptive of what one wants to do. Just seeing that, someone could interpret it as a number not a boolean.

Reply:
Resolved by flipping logic. Default is now off, `1` and `true` set to enabled.